tests/core/tools/color.js

```diff
@@ -0,0 +1,113 @@
+/* bender-tags: editor */
+( function() {
+	'use strict';
+
+	bender.test( {
+		setUp: function() {
+			this.createColor = function( colorCode ) {
+				return new CKEDITOR.tools.style.Color( colorCode );
+			};
+		},
+
+		'test color object creation': function() {
+			var colorObject = this.createColor( '' );
+
+			assert.isObject( colorObject );
+		},
+
+		'test color from invalid string color name return default color': function() {
```

Comment:
```suggestion
		'test color from invalid string color name returns default color': function() {
```

Please remember about proper form :wink:

Reply:
Correct in all test case names.